x-pack/legacy/plugins/uptime/public/components/functional/ping_list/ping_list.tsx

```diff
@@ -220,120 +246,98 @@ export const PingListComponent = ({
         return (
           <EuiButtonIcon
             onClick={() => toggleDetails(item, itemIdToExpandedRowMap, setItemIdToExpandedRowMap)}
-            disabled={!item.error && !(item.http?.response?.body?.bytes > 0)}
+            disabled={!item.error && !(item.http?.response?.body?.content_bytes ?? 0 > 0)}
             aria-label={
-              itemIdToExpandedRowMap[item.id]
+              itemIdToExpandedRowMap[item.monitor.id]
```

Comment:
shouldn't this also be changes to '@timestamp'?

Reply:
++ you are right, will fix.

Reply:
Addressed in https://github.com/elastic/kibana/pull/59392/commits/433bb7475a367ec6b573e7fadc94ce578733d468.